tasks/eu/geo.py

```diff
@@ -56,6 +56,7 @@ def output(self):
 
 
 class ImportSHNGeoms(Shp2TempTableTask):
+    level = IntParameter()
```

Comment:
Why is this change needed?

Reply:
In EU we have three different simplifications for the three existing levels (see [here](https://github.com/CartoDB/bigmetadata/blob/08e5b5fafe65f0469bc108c83b0e34ac4d76eab9/tasks/simplifications.json#L248-L271)). We need a to identify the level with the task id